Godunov/hydro_godunov.H

```diff
@@ -82,8 +84,9 @@ void ComputeEdgeState ( amrex::Box const& bx, int ncomp,
                         amrex::Real dt,
                         amrex::BCRec const* d_bcrec,
                         int const* iconserv,
-                        const bool use_ppm, bool is_velocity,
-                        const bool use_forces_in_trans);
+                        const bool use_ppm, const bool use_forces_in_trans,
```

Comment:
note I swapped the header file argument names. since it's two bools in a row anyone that looked at the header file for the interface might have accidentally swapped them

Reply:
just a typo in the header file, I doubt anyone called this function since we call it internally to amrex-hydro